### PR TITLE
Update sequel-pro-nightly

### PR DESCRIPTION
**Pull request by RJHsiao**
As title

### CODE REVIEW

File: Casks/sequel-pro.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  name 'Sequel Pro'
+  homepage 'http://www.sequelpro.com/'
+  license :mit
+
+  depends_on :macos => '>= :leopard'
+
+  app 'Sequel Pro.app'
+
+  zap :delete => '~/Library/Application Support/Sequel Pro/Data'
+end
```
Comment (vitorgalvao) [inline Casks/sequel-pro.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

In this case, [`sequel-pro` is already present in the main repo](https://github.com/caskroom/homebrew-cask/blob/master/Casks/sequel-pro.rb). You need to remove it.

**Comment by RJHsiao** [comment]
@vitorgalvao  
OK, I will separate this PR later.  
But can I update sequel-pro in the main repo even it's a RC release, not a stable release?

**Comment by vitorgalvao** [comment]
Since it is an important fix, yes, no sense in having a new cask for a temporary fix.

**Comment by RJHsiao** [comment]
Hi @vitorgalvao, I re-push it with change Casks/sequel-pro-nightly.rb only, please check it.

**Pull request merged**